src/OrchardCore.Modules/OrchardCore.Email/Drivers/EmailSettingsDisplayDriver.cs

```diff
@@ -71,7 +72,7 @@ public override async Task<IDisplayResult> EditAsync(EmailSettings settings, Bui
         .OnGroup(EmailSettings.GroupId);
     }
 
-    public override async Task<IDisplayResult> UpdateAsync(EmailSettings settings, UpdateEditorContext context)
+    public override async Task<IDisplayResult> UpdateAsync(ISite site, EmailSettings settings, IUpdateModel updater, UpdateEditorContext context)
```

Comment:
Why are these changes necessary? Also in a couple of other drivers.

Reply:
it was necessary to add the signal on the ISite object. This is not longer the case so this change it no longer needed